src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/response.tag

```diff
@@ -45,6 +52,43 @@
       </c:choose>:
   </div>
   <c:choose>
+    <c:when test="${questionWithResponses.feedbackParticipantCommentsOnResponsesAllowed}">
```

Comment:
Can just put another <c:choose> otherside the <c:choose>.

Reply:
Didn't quite understand this comment. I updated this file.

Reply:
Can we put the change other the whole
```
<c:choose>
    <c:when test="${questionWithResponses.question.questionTypeConstsum}">
      ${response.submissionFormHtml}
    </c:when>
    <c:otherwise>
      <div class="${divClassType}">
        ${response.submissionFormHtml}
      </div>
    </c:otherwise>
  </c:choose>
```
?

Reply:
Maybe not because comment forms has to be put in same division as ` ${response.submissionFormHtml}`?

Reply:
Why? `evalueeForm` is per response.

Reply:
Right. I have made the changes. Please confirm if it is okay before I make God Mode changes?

Reply:
You are doing good 👍